activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -316,4 +316,5 @@ def set_defaults(target) # :doc:
   end
 end
 
+# :stopdoc:
 HashWithIndifferentAccess = ActiveSupport::HashWithIndifferentAccess
```

Comment:
Removing from the side bar means that `HashIwthIndifferentAccess` is not part of the public API and is only internal. People will not find this constant anymore documented, only the one inside `ActiveSupport`.